scripts/qa_crowbarsetup.sh

```diff
@@ -3594,7 +3594,16 @@ function oncontroller_magnum_service_setup ()
         # os-distro property
         openstack image create --file $service_image_name \
             --disk-format qcow2 --container-format bare --public \
-            --property os_distro=fedora-atomic magnum-service-image
+            --property hw_firmware_type=uefi \
+            --property os_distro=jeos magnum-service-image
+    fi
+
+    if ! nova flavor-show m1.magnum; then
```

Comment:
The flavors are already created. See https://github.com/SUSE-Cloud/automation/commit/78d9659b04a222f94f01d46000be8d6226b62074 . Sorry - haven't seen that you create the flavors here, too.